AppDevAnnouncements/Classes/NotificationViewController.swift

```diff
@@ -70,3 +58,10 @@ public class NotificationViewController: UIViewController {
     }
 
 }
+
```

Comment:
can we add a `// MARK: - UIViewController+Extension to present notification view controller`